lib/core/Manager.js

```diff
@@ -481,6 +482,23 @@ Manager.prototype._parseDependencies = function (decEndpoint, pkgMeta, jsonKey)
         childDecEndpoint.unresolvable = !!decEndpoint.unresolvable;
 
         // Otherwise, just fetch it from the repository
+        
+        //Adjust dependency source if relative, considering its own configuration, if any
+        if (/^(\.|\.\.)\//g.test(childDecEndpoint.source)) {
```

Comment:
I do not think this will work on win32 systems due to the `/` and relative path correct?